statsmodels/tsa/statespace/tests/results/test_ucm.R

```diff
@@ -94,12 +94,23 @@ print(exp(res_cycle$optim.out$par)) # [37.57197224]
 print(res_cycle$optim.out$value)    # 672.3102588
 
 # Seasonal (with fixed period=4, seasonal variance=0.1)
-mod_seasonal <- SSModel(dta$unemp ~ -1 + SSMseasonal(4, Q=matrix(0.1), sea.type='dummy', P1=diag(1)*1e6), H=matrix(NA))
+# Note: matching this requires setting loglikelihood_burn=0
+mod_seasonal <- SSModel(dta$unemp ~ -1 + SSMseasonal(4, Q=matrix(0.1), sea.type='dummy', P1=diag(rep(1e6, 3))), H=matrix(NA))
```

Comment:
I noticed that the first P1 initialization is 1e6 oo', where o is the vector of ones of length 3.  I think we wanted the diagonal version.